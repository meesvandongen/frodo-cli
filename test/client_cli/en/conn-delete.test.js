import cp from 'child_process';
import { promisify } from 'util';

const exec = promisify(cp.exec);
const CMD = 'frodo connections delete --help';
const { stdout } = await exec(CMD);

test("CLI help interface for 'delete' should be expected english", async () => {
  // Arrange
  expect(stdout).toMatchSnapshot();
});
