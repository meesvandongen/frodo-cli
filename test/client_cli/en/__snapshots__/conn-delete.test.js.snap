// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CLI help interface for 'delete' should be expected english 1`] = `
"Usage: frodo conn delete [options] <host>

Delete connection profiles.

Arguments:
  host        Access Management base URL, e.g.: https://cdk.iam.example.com/am.
              To use a connection profile, just specify a unique substring.

Options:
  --verbose   Verbose output during command execution. If specified, may or may
              not produce additional output.
  --debug     Debug output during command execution. If specified, may or may
              not produce additional output helpful for troubleshooting.
  -h, --help  Help
"
`;
